### ISSUE TITLE
The current process for filtering is laborious.

### ISSUE THREAD

**Issue by Pixeltramp** [AHAAAAAAA/PokemonGo-Map#1429]
Any chance you could restore the comma-delineated flag, or allow a quicker input method like a frontend-based comma-delineated input?

**Comment by 2-m** [comment]
Agreed

**Comment by talentedmrripley** [comment]
I would like to see this return also.

**Comment by FaustKnight** [comment]
Agreed, comma-delineated would be amazing. Best if it can also be put in a config file and not just the map.
